cache.js

```diff
@@ -0,0 +1,66 @@
+const MAX_CACHE_SIZE = 100
+
+const generateDefKey = (data, to) => {
+  if (
+    !data ||
+    data.length === 0 ||
+    (!isArrayBuffer(data) && typeof data !== "string")
+  ) {
+    return;
+  }
+
+  const toStr = !!to ? Buffer.from(to).toString("hex") : "";
+
+  let dataStr = data.toString("hex");
```

Comment:
Didn't we want to do `Buffer.from(data).toString('hex')` in the event of `data` being a u8array?

Reply:
Right. I pulled out the string handling in to a separate function that we can use for both `to` and `data` since we were starting to perform similar changes to both. [e07a48f ](https://github.com/GridPlus/eth-lattice-keyring/commit/e07a48f392203cb888338402fe1340530cb0522b)